### ISSUE TITLE
.net core not working out of the box

### ISSUE THREAD

**Issue by Minikloon** [RogueException/Discord.Net#294]
This is my app's project.json:
```
{
  "version": "1.0.0-*",
  "buildOptions": {
    "emitEntryPoint": true
  },

  "dependencies": {
    "Microsoft.NETCore.App": {
      "type": "platform",
      "version": "1.0.0"
    },
    "Newtonsoft.Json": "9.0.1",
    "Discord.Net": "0.9.4"
  },

  "frameworks": {
    "netcoreapp1.0": {
      "imports": "dnxcore50"
    }
  }
}
```
I'm getting this error:
```
log  : Restore failed in 1070ms.
Errors in C:\Development\Csharp\HackSlab\src\HackSlab\HackSlab.xproj
    Package Nito.AsyncEx 3.0.1 is not compatible with netcoreapp1.0 (.NETCoreApp,Version=v1.0). Package Nito.AsyncEx 3.0.1 supports:
      - monoandroid (MonoAndroid,Version=v0.0)
      - net40 (.NETFramework,Version=v4.0)
      - net45 (.NETFramework,Version=v4.5)
      - netcore45 (.NETCore,Version=v4.5)
      - portable-net40+netcore45+sl4+wp71 (.NETPortable,Version=v0.0,Profile=net40+netcore45+sl4+wp71)
      - portable-net40+sl5+win8+wp8+wpa81 (.NETPortable,Version=v0.0,Profile=Profile328)
      - portable-net40+win8 (.NETPortable,Version=v0.0,Profile=Profile5)
      - portable-net45+win8+wp8+wpa81 (.NETPortable,Version=v0.0,Profile=Profile259)
      - portable-net45+win8+wpa81 (.NETPortable,Version=v0.0,Profile=Profile111)
      - sl4 (Silverlight,Version=v4.0)
      - sl5 (Silverlight,Version=v5.0)
      - wp71 (WindowsPhone,Version=v7.1)
      - wp8 (WindowsPhone,Version=v8.0)
      - wpa81 (WindowsPhoneApp,Version=v8.1)
      - xamarinios10 (Xamarin.iOS,Version=v1.0)
      - xamarinmac20 (Xamarin.Mac,Version=v2.0)
    One or more packages are incompatible with .NETCoreApp,Version=v1.0.
```

I tried setting my app to netcore45, but then it says Discord.Net doesn't support netcore45.

**Comment by RogueException** [comment]
This is an issue in 0.9.x that has already been solved in 1.0.

You can work around this issue by changing the imports section of your project.json to:
```json
"imports": [
  "dnxcore50",
  "portable-net45+win8"
]
```

**Comment by Minikloon** [comment]
Neat, thanks.
